### ISSUE TITLE
Feature suggestion: Add Ethernet support

### ISSUE THREAD

**Issue by chetan51** [chetan51/sidestep#10]

**Comment by digitalquery** [comment]
This would be amazingly useful.

Actually, supporting any non-wireless ethernet networking device would be really useful - sidestep doesn't work for my 3G dongle either.

**Comment by pajp** [comment]
I agree - Sidestep is a super easy way of setting up an ad-hoc VPN between two Macs using Back To My Mac (you can run SSH between two BTMM-enabled Macs), but at the moment it only works if you are connected using Wi-Fi.

**Comment by devalnor** [comment]
I vote for !

**Comment by emich** [comment]
+1

**Comment by twobitlabs** [comment]
I love Sidestep and use it all the time as a general purpose VPN client. It's just a bummer that I have to switch to wireless to get it to work. Using Mountain Lion and over ethernet it looks like it's working but doesn't actually reroute traffic.

**Comment by gregdel** [comment]
+1
Ethernet support would be nice !

**Comment by aascensao** [comment]
+1! Ethernet support would be a great feature to complement this good app!

**Comment by naggie** [comment]
Yes! For arbitrary interfaces. Perhaps it could detect the active interface. Then I can use it with USB tethering.

**Comment by mikeyarce** [comment]
+1 this would be great to have!
